src/Server.py

```diff
@@ -214,7 +214,7 @@ def build_trie(self):
         def dfs(node):
             prefix, isword, count = node['name'], node['isword'], node['count']
             if isword:
-                self.__insert(prefix, isword, from_db=True, count=count)
+                self.__insert(prefix, isword=True, from_db=True, count=count)
```

Comment:
nit: not sure but do we want to use (**kwargs) here ?

Reply:
I changed this because the editor detected that the `__insert` method has only one positional argument now, when we refactored that function must have forgotten to update the reference in this method.